src/app/formatxml.py

```diff
@@ -143,7 +152,9 @@ def pretty(node, level):
         text = text.replace('&', '&amp;').replace('>', '&gt;').replace('<', '&lt;')
         ser += text
     for child in node:
-        ser += pretty(child, child_level)
+        res = pretty(child, child_level)
+        if not(IS_EXCEPTION and 'standardLicenseHeader' in res):
```

Comment:
This needs to checked in the context of `pretty` func not at a global level I think.

Reply:
Done